Sources/FirestoreManager/FirestoreManager.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  FirestoreManager.swift
+//  
+//
+//  Created by Anna Münster on 26.08.21.
+//
+
+import Firebase
+import FirebaseFirestoreSwift
+
+
+open class FirestoreManager {
+    static let database = Firestore.firestore()
```

Comment:
As there is is also Firebase Database (Realtime Database), I think renaming this to "firestore" would be better.